flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestFlinkUpsert.java

```diff
@@ -147,7 +147,7 @@ public void testUpsertAndQuery() {
           tableName);
 
       List<Row> rowsOn20220301 =
-          Lists.newArrayList(Row.of(2, "b", dt20220301), Row.of(1, "a", dt20220301));
+          Lists.newArrayList(Row.of(1, "b", dt20220301), Row.of(2, "b", dt20220301));
```

Comment:
We may be need to modify the ut data to match the purpose of the test.
Let the second sql `upsert` the data in the first.

Reply:
thanks for your suggestion. I have verified the order of row and re-submitted the code.